### ISSUE TITLE
catalog should not be downloaded on every startup

### ISSUE THREAD

**Issue by bridiver** [brave-intl/bat-native-ads#140]
from https://github.com/brave-intl/bat-native-ads/pull/139 catalog is downloaded on browser startup, but that's a problem for privacy. The reasoning for token redemption also doesn't seem valid because many people (myself included) keep the browser open for weeks at a time. It can happen immediately when ads is enabled, but after that it should happen on a randomized schedule that is independent of browser startup/shutdown.

**Comment by bridiver** [comment]
there is no security/privacy review label here so cc @diracdeltas

**Comment by bridiver** [comment]
there is also the opposite issue if you frequently open/close the browser during the day

**Comment by tmancey** [comment]
This issue fix a bug where the catalog was downloaded twice on startup. The logic where the catalog is downloaded on browser startup is legacy from Muon codebase so we need to decide if this is an issue

**Comment by jsecretan** [comment]
I don't think this is currently a problem, because the catalog is not only downloaded on startup, but on 2 hour intervals with 10% jitter.

**Comment by bridiver** [comment]
so as discussed on Slack I think the most urgent problem is that there is no minimum time between downloads so successive restarts of the browser would trigger frequent catalog downloads, but if anything the 2 hour interval makes it even easier to track browser usage because we're getting frequent pings that will stop when the browser is closed.

**Comment by diracdeltas** [comment]
given the frequency of this ping, i think it's valid to require more variance in the pings and avoid it on restarts
